lib/network_ssl/network_ssl.c

```diff
@@ -480,6 +476,9 @@ network_ssl_read(struct network_ssl_ctx * ssl, uint8_t * buf,
     int (* callback)(void *, ssize_t), void * cookie)
 {
 
+	/* We must have a root certificate set up. */
```

Comment:
`/* We must be initialized before we can get here. */`

I suspect your comment came from when we were having `_loadroot` as something which was called from upstream.

Reply:
Technically, I'd say that
```
/* We must be initialized before we can get here. */
```
rather than
```
/* We must be initialized before we can get here. */
```

I mean, the whole point of the `assert` is that we **can** get there without being initialized.  :)

Reply:
Umm, what?

Reply:
argh.  This is what I'd like:
```
/* We must be initialized before we get here. */
```